crosscorrelation.Rmd

```diff
@@ -0,0 +1,384 @@
+---
+title: "uploadid"
+author: "Amenze"
+date: "June 28, 2018"
+output: html_document
+---
+
+```{r}
+setwd("/Users/Amenze/Desktop/tidepool/refdata")
+```
+
+
+```{r }
```

Comment:
Could you add a comment to each of these with a link to the documentation and a one sentence synopsis of why the lib is necessary?

Reply:
ok. will upload all changes